catalog/dags/database/batched_update/batched_update_dag.py

```diff
@@ -139,8 +139,8 @@
             pattern="^SET updated_on = NOW()",
         ),
         "additional_where": Param(
-            default="",
-            type="string",
+            default=None,
+            type=["null", "string"],
```

Comment:
Very interesting that specifying a type is used to make a field required and allowing null values is the mechanism to make a param optional. Not super explicit, Airflow!

Reply:
It actually relies on `jsonschema` here, there's no documentation about it in Airflow at all 😅

Reply:
I did come across this while reviewing the PR: 

https://airflow.apache.org/docs/apache-airflow/stable/core-concepts/params.html#use-params-to-provide-a-trigger-ui-form

Reply:
Ah great point! I think I was specifically thinking about the `pattern` parameter here which is _only_ in the JSON schema docs, and misattributed that to the docs missing entirely 😅